src/components/search/TagSearchBar.tsx

```
@@ -140,17 +141,17 @@ export const TagSearchBar: React.FC<TagSearchBarProps> = ({
         >
           {tagSuggestions.map((tag, index) => (
             <li
-              key={tag}
-              id={`tag-suggestion-${tag}`}
+              key={tag.name}
+              id={`tag-suggestion-${tag.name}`}
               role="option"
               aria-selected={index === activeIndex}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<option>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)